homeassistant/components/sharkiq/strings.json

```diff
@@ -20,7 +20,7 @@
       "unknown": "[%key:common::config_flow::error::unknown%]"
     },
     "abort": {
-      "already_configured_account": "[%key:common::config_flow::abort::already_configured_account%]",
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]",
```

Comment:
```suggestion
```
`already_configured_account`
not used in
`config_flow.py` and `test_config_flow.py`

Reply:
If it's not used, we should just remove it.